streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamOuterJoinTest.java

```diff
@@ -53,15 +55,16 @@
 
 import static java.time.Duration.ZERO;
 import static java.time.Duration.ofMillis;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@Category(IntegrationTest.class)
 public class KStreamKStreamOuterJoinTest {
```

Comment:
I believe this test was not running before due to the lack of @Category annotation.